monkey/infection_monkey/exploit/mssqlexec.py

```diff
@@ -25,18 +25,31 @@ class MSSQLExploiter(HostExploiter):
     # Time in seconds to wait between MSSQL queries.
     QUERY_BUFFER = 0.5
     SQL_DEFAULT_TCP_PORT = '1433'
+
     # Temporary file that saves commands for monkey's download and execution.
     TMP_FILE_NAME = 'tmp_monkey.bat'
+    TMP_DIR_PATH = "C:\\windows\\temp\\monkey_dir"
```

Comment:
This will fail if Windows is not installed on C, even though it could work

Reply:
maybe we can do this for default path?
`import os
winpath = os.environ['WINDIR']`

Reply:
This might run on Linux, so no. You need to have this run as part of the payload. Or start by saving to %temp%\\monkey_dir

Reply:
We should do the same for dropper:
![image](https://user-images.githubusercontent.com/36815064/63929886-e804b200-ca5a-11e9-9902-9a90506e244c.png)

Reply:
Yes, see #350 for where this screws us